lib/models/event.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:convert';
+
+import 'package:date_format/date_format.dart';
+
+class Event {
+  final int eid;
+  final String title;
+  final String description;
+  final String date;
+  final bool attending;
+  final int numattenders;
+
+  Event({
+    this.eid,
+    this.title,
+    this.description,
+    this.date,
+    this.attending,
+    this.numattenders,
+  });
+
+  factory Event.fromJson({
+    Map<String, dynamic> map,
+    int creatorUid,
```

Comment:
extraneous creatorUid; also only the school creates events

Reply:
this was left over from before i just kept it in